backend/src/v1/services/report-calc-service.ts

```diff
@@ -110,11 +110,7 @@ accessed as follows:
   const medianBonusPayFemale = bonusPayStats.getMedian("female");
 */
 class GroupedColumnStats {
-  static readonly REF_CATEGORY_PREFERENCE = [
-    GENDER_CODES.MALE,
-    GENDER_CODES.UNKNOWN,
-    GENDER_CODES.NON_BINARY,
-  ];
+  static readonly REF_CATEGORY_PREFERENCE = [['M'], ['U'], ['X']];
```

Comment:
Was there any need to change this?

Reply:
<img width="592" alt="Screenshot 2024-03-25 at 3 24 27 PM" src="https://github.com/bcgov/fin-pay-transparency/assets/4842061/2ef9c756-5913-48c6-b3fb-214933fe973d">
Somehow, this is causing some runtime errors